src/utils/tdex.ts

```diff
@@ -27,14 +28,12 @@ function tdexOrdersForProvider(pair: Pair, providerEndpoint: string, markets: Ma
   return orders;
 }
 
-const TDEX_REGISTRY_URL = 'https://raw.githubusercontent.com/TDex-network/tdex-registry/master/registry.json';
-
 /**
  * Get a list of registered providers from TDEX_REGISTRY_URL
  * @returns a list of providers
  */
 export async function getProvidersFromRegistry(): Promise<TDEXProvider[]> {
-  const res = (await axios.get(TDEX_REGISTRY_URL)).data
+  const res = (await axios.get(await getRegistryURL())).data
```

Comment:
should this also propagate the network from the rop

Reply:
Same issue, is used inside a module imported by the Svelte component, not sure on how to pass params from the Svelte component to the module when importing.

Reply:
what if in `src/stores/tdexstore`  you pass network in the `fetchProvidersAndMarketsAndSet` called by the `marina.on('NETWORK')`

Reply:
Commit 99635d4 should fix this.